cmd/shell.go

```diff
@@ -209,42 +209,49 @@ func FailOnError(err error, msg string) {
 func AmqpChannel(conf Config) (*amqp.Channel, error) {
 	var conn *amqp.Connection
 
-	if conf.AMQP.SSL != nil {
-		if strings.HasPrefix(conf.AMQP.Server, "amqps") == false {
-			err := fmt.Errorf("SSL configuration provided, but not using an AMQPS URL")
+	if conf.AMQP.TLS == nil {
+		// Configuration did not specify SSL options
+		conn, err := amqp.Dial(conf.AMQP.Server)
+		if err != nil {
 			return nil, err
 		}
 
-		cfg := new(tls.Config)
+		return conn.Channel()
```

Comment:
It seems like instead of having a return here you could have an if/else, right? Thus factoring out the conn.Channel()